fsspec/implementations/local.py

```diff
@@ -63,51 +63,49 @@ def glob(self, path, **kwargs):
         return super().glob(path, **kwargs)
 
     def info(self, path, **kwargs):
+        t = None
         if isinstance(path, os.DirEntry):
             # scandir DirEntry
             out = path.stat(follow_symlinks=False)
             link = path.is_symlink()
+            if link:
+                t = "link"
```

Comment:
Looking at `smb` and `sftp` I assumed `link` type is desired. This kind of makes `islink` field obsolete, but I guess removing it would result in breaking the api.

Reply:
@pared But how can I now tell that this link is pointing to a file or a directory?